### DISCUSSION TITLE
Templater actions available in another vault. How is this possible?

### DISCUSSION THREAD

**Discussion by gala8y** [SilentVoid13/Templater#769] [Help]
Edited copy of my post [0] on Obsidian Members Group Discord from two days ago:

I just created a little tiny, empty vault within my main vault. This new vanilla vault is empty with no plugins. Oddly enough, it uses templater settings from main vault while creating new note (front matter gets populated exactly as intended in the main vault). It naturally happens only when main vault is open. 

[0] https://discord.com/channels/686053708261228577/840286238928797736/995998183433965618

I can still reproduce this. Is this Templater issue or Obsidian issue?

**Comment by welpdx** [comment]
I couldn't replicate issue sry. I made a new vault via `app.openVaultChooser()` within an existing vault. The new vault didn't have shortcut settings as the main. I downloaded and installed templater from the community plugins. It has default settings

**Comment by gala8y** [comment]
My 'Add new' template looks like this:

```
---
birth : <% tp.date.now("YYYY-MM-DDTHH:mm") %>
aliases : [  ]
type : 
---
```
 and absolutely gets executed in the new, vanila vault (which is inside of one of folders of the main vault).

**Comment by welpdx** [comment]
When I create a new vault in another vault (that already has templater and other settings in it), it is exactly like a new fault for me. I have to download Templater from the `community plugins` in order to run  your template. 
Are you perhaps using the Obsidian Default Template plugin? Can you take a screen shot or something of your new vault.

By the way, this is what my directory looks like when I create a new vault inside another vault:
```ascii
C://
├─ OriginalVault/
│  ├─ .obsidian/
│  │  ├─ plugins/
│  │  │  ├─ templater-obsidian/
│  ├─ someFile.md
│  ├─ innerVault/
│  │  ├─ .obsidian/
```

As you can see, after creation the inner vault makes its own `.obsidian/` folder which contains the default obsidian vault settings. I would need to download and enable Templater to have the `plugins/templater-obsidian/` directory showing up with all the Templater Settings.

**Comment by gala8y** [comment]
Thx for checking it. It's odd, that's why I am posting here. I know how this works (seperate .obsidian folder, ...).
If I switch Obsidian built-in Templates plugin off in main and new vaults, I still get front matter inserted in new notes in new vault (if main vault is open).

**Comment by welpdx** [comment]
Sorry for all that, just wanted to ask some clarifying questions 'cause what you are experiencing is indeed odd.

**Comment by gala8y** [comment]
Absolutely no need to be sorry, I think we are just trying to be precise.
